FlagEditor/gitTools/git-config-path.js

```diff
@@ -0,0 +1,35 @@
+/*!
```

Comment:
is there a specific reason these are included directly vs used as npm dependencies?

Reply:
I was looking to minimize the overall # of dependencies that are not **extremely** widely used, and it seemed to me that anything that people explicitly install to interface with git at the filesystem level as the expected behavior for the package was at a much higher-than-usual risk for supply chain attacks (attackers could expect that this code will run directly on privileged machines that have developers credentials on the filesystem in many cases).  Maybe my risk->reward judgement here is wrong, I'm happy to bring them in directly :)